cmd/antrea-controller/controller.go

```diff
@@ -92,7 +92,8 @@ func run(o *Options) error {
 	networkPolicyInformer := informerFactory.Networking().V1().NetworkPolicies()
 	nodeInformer := informerFactory.Core().V1().Nodes()
 	cnpInformer := crdInformerFactory.Security().V1alpha1().ClusterNetworkPolicies()
-	externalEntityInformer := crdInformerFactory.Core().V1alpha1().ExternalEntities()
+	externalEntityV1Informer := crdInformerFactory.Core().V1alpha1().ExternalEntities()
```

Comment:
I'm not sure the common convention of CRD version evolution, is it common to keep previous CRD compatible even when it's in alpha phase? I think the benefit of alpha is we can improve the feature without the burden of a stable one, like the extra code in this PR for version conversion. https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages

Reply:
@abhiraut and I would agree, the tricky part is that v1alpha1 ExternalEntity users have already tagged a milestone release with Antrea v0.10, and @suwang48404 wanted our change to still be compatible at this point. Maybe we could remove v1alpha1 ExternalEntity support in the next next release once all existing EE consumers switch to v1alpha2?

Reply:
@suwang48404 since antreaplus is tagged with v0.10, we will simply update the existing ExternalEntity spec? Let me know if you are not sure what the implications are.